playbooks/byo/rhel_subscribe.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- include: ../common/openshift-cluster/evaluate_groups.yml
+  vars_files:
+  - ../../byo/openshift-cluster/cluster_hosts.yml
```

Comment:
k, let me refresh my understanding of that change, i thought this was the new way